test/geometry/epipolar/test_fundamental.py

```diff
@@ -100,6 +100,15 @@ def test_shape_noweights(self, batch_size, num_points, device, dtype):
         F_mat = epi.find_fundamental(points1, points2, weights)
         assert F_mat.shape == (B, 3, 3)
 
+    @pytest.mark.parametrize("batch_size", [1, 2, 3])
```

Comment:
Worth to try an unbatched version here.
https://github.com/kornia/kornia/blob/a0c6f8d1776501e34489590932aa712d75cb35cc/test/geometry/liegroup/test_so3.py#L43

Reply:
I guess it would be better  to make changes to all the functionalities within the geometry or epipolar module together in a single PR